app/assets/appfilter.xml

```diff
@@ -5364,8 +5365,8 @@
   <item component="ComponentInfo{com.pittvandewitt.viperfx/com.pittvandewitt.viperfx.MainActivity}" drawable="viper4android" name="ViPER4Android FX" />
   <item component="ComponentInfo{com.pittvandewitt.viperfx/com.pittvandewitt.viperfx.main.StartActivity}" drawable="viper4android" name="ViPER4Android FX" />
   <item component="ComponentInfo{com.pittvandewitt.viperfx/com.pittvandewitt.viperfx.StartActivity}" drawable="viper4android" name="ViPER4Android FX" />
-  <item component="ComponentInfo{com.wstxda.viper4android/com.wstxda.viper4android.MainActivity}" drawable="viperfx" name="ViperFX" />
-  <item component="ComponentInfo{com.aam.viper4android/com.aam.viper4android.MainActivity}" drawable="viperfx" name="ViperFX" />
+  <item component="ComponentInfo{com.wstxda.viper4android/com.wstxda.viper4android.MainActivity}" drawable="viper4android" name="ViperFX" />
+  <item component="ComponentInfo{com.aam.viper4android/com.aam.viper4android.MainActivity}" drawable="viper4android" name="ViperFX" />
```

Comment:
`viper4android` was not added in this PR, so you essentially just broke that icon

Reply:
Yes, both are icons for the same app, just different, updated versions.